psi4/driver/qcdb/libmintscoordentry.py

```diff
@@ -295,6 +295,10 @@ def mass(self):
         """The atomic mass of the current atom."""
         return self.PYmass
 
+    def set_mass(self, mass):
+        """Assign the mass of the atom (useful for isotopic substitutions)."""
+        self.PYmass = mass
```

Comment:
length? this is for one atom. But yeah, I'll add a >0.0 check.

Reply:
Yeah, I'll add a few sanity checks. But primarily this is for isotopic tweaking, not routine use. And masses coming directly from Molecule instantiations are checked w/i an inch of their lives.

Reply:
It might be good to add this to the docs for this class in this case. When we remodel classes its good to try and apply general Python best practices. We range between the best practices extremes for good reasons, but when folks are looking to build new tech and modeling from current classes it would be good to have them marked.

Reply:
Indeed. `Molecule` is getting its docstring upgrades as-you-go. And most should happen in qcdb/qcdb to avoid duplication.